catalog/task/prepare-skopeo-image-string/0.1/README.md

```diff
@@ -0,0 +1,28 @@
+# prepare-skopeo-image-string
+
+Tekton task to create a string from a snapshot that can be passed to skopeo.
+
+The purpose of this task is to create a string that can be passed as an argument to skopeo inspect.
+The task creates this string by extracting the containerImage value from the snapshot provided and
+prepending 'docker://' to it.
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| snapshot | The Snapshot in JSON format containing a single container image | No | - |
```

Comment:
Noob question: Are multi-component snapshots not supported? Is that something that will come later?

Reply:
@scoheb is this correct?

Reply:
ok, thanks.

Reply:
As discussed in standup, this is not a correct assumption. I will open new PRs to fix that